### PR TITLE
doc: Remove default issue titles

### PR DESCRIPTION
**Pull request by bitcrazed**
Removes default issue titles as per today's discussion.

### CODE REVIEW

**Comment by DHowett-MSFT** [comment]
nit: pretend your pull request titles complete the sentence `This pull request will...`

File: .github/ISSUE_TEMPLATE/Bug_Report.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-name: Bug report 🐛
+name: "Bug report \U0001F41B"
```
Comment (zadjii-msft) [inline .github/ISSUE_TEMPLATE/Bug_Report.md]: why'd this go away? I miss my 🐛 friend

File: .github/ISSUE_TEMPLATE/Guidance_Issue.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 ---
 name: Community Guidance Request ✨
```
Comment (zadjii-msft) [inline .github/ISSUE_TEMPLATE/Guidance_Issue.md]: but the ✨ boi survived?

File: .github/ISSUE_TEMPLATE/Documentation_Issue.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-name: Documentation Issue 📚
+name: "Documentation Issue \U0001F4DA"
```
Comment (zadjii-msft) [inline .github/ISSUE_TEMPLATE/Documentation_Issue.md]: oh no they're all ded

File: .github/ISSUE_TEMPLATE/Bug_Report.md (hunk 1 above)
Comment (bitcrazed) [reply .github/ISSUE_TEMPLATE/Bug_Report.md]: Looks like Github translated bug emoji into unicode. Looks like it's the correct codepoint for bug, tho.

File: .github/ISSUE_TEMPLATE/Guidance_Issue.md (hunk 2 above)
Comment (bitcrazed) [reply .github/ISSUE_TEMPLATE/Guidance_Issue.md]: Didn't need to modify this template so I think GH left the emoji alone.

Comment (bitcrazed) [reply .github/ISSUE_TEMPLATE/Guidance_Issue.md]: Yeah, odd. Something must've happened in the GH UX.

File: .github/ISSUE_TEMPLATE/Guidance_Issue.md [hunk 4]
```diff
@@ -1,9 +1,10 @@
...
-about: Suggest somewhere the Windows Terminal Team needs to provide community guidance through new documentation or process.
-title: "Guidance"
+about: Suggest somewhere the Windows Terminal Team needs to provide community guidance
+  through new documentation or process.
+title: ''
 labels: Issue-Docs
-assignees: 'bitcrazed'
```
Comment (bitcrazed) [inline .github/ISSUE_TEMPLATE/Guidance_Issue.md]: Also don't think I specifically changed default assignees for this template, but GH changed to unquoted.

Comment (bitcrazed) [reply .github/ISSUE_TEMPLATE/Guidance_Issue.md]: Cinnamon_msft is now default assignee for guidance docs ;)

**Comment by DHowett-MSFT** [comment]
Now that I think about it, I recall us saying "Guidance never worked right, we should delete it." What do you think? None of our users have used it properly. They just use it to ask for docs.

**Comment by bitcrazed** [comment]
Good point. Deleting now.

**Pull request merged**